web/third/bootstrap/css/bootstrap-btn.css

```diff
@@ -209,39 +209,11 @@ fieldset[disabled] .btn-success.active {
 .btn-info:hover,
 .btn-info:focus,
 .btn-info:active,
-.btn-info.active,
-.open .dropdown-toggle.btn-info {
+.btn-info.active {
```

Comment:
Do we use both `btn-info` and `active` class together anywhere? I cannot find one.

Reply:
Ahh yeah this line should be removed. I couldn't find one as well.